react/features/reactions/components/native/RaiseHandButton.js

```diff
@@ -170,8 +173,7 @@ function _mapStateToProps(state): Object {
     return {
         _enabled: enabled,
         _localParticipant,
-        _raisedHand: hasRaisedHand(_localParticipant),
-        _styles: ColorSchemeRegistry.get(state, 'Toolbox').raiseHandButton
```

Comment:
Can you drop that `raisedHandButton` style from the registry?

Reply:
I removed it. It already appears in the PR.

Reply:
👍